examples/src/indexes/vector_stores/prisma_vectorstore/prisma/migrations/00_init/migration.sql

```diff
@@ -3,6 +3,7 @@ CREATE EXTENSION IF NOT EXISTS vector;
 CREATE TABLE "Document" (
     "id" TEXT NOT NULL,
     "content" TEXT NOT NULL,
+    "namespace" TEXT DEFAULT 'default',
```

Comment:
Just making sure, this won't break legacy users right? It's completely optional?